policies/new_chapter_policy.md

```diff
@@ -0,0 +1,136 @@
+| Version  | 1.0                                        |
+| Title    | New PyLadies Chapter Policy                |
+| Authors  | Mariatta <mailto:mariatta@pyladies.com>    |
+| Status   | Draft                                      |
+
+                                
+# TL;DR:
+
+The policy should answer the following questions
+
+1. Why is the policy necessary
+2. What is the purpose of a PyLadies chapter? What does it mean to start a PyLadies chapter? 
+3. What are minimum responsibilities and expectation when someone requested a PyLadies chapter?
+4. What kind of support can new chapter leaders expect from Global PyLadies leadership team?
+5. What questions do we ask in our form?
+6. How do we review new chapter requests?
+7. For accountability, how do we document new chapter creations?
+
+Timeline: I'd like to have this new process in place by end of January 2020.
+
+# Details
+
+## 1. Why is this policy necessary?
+
+- We had many inactive chapters, that never took off since they were created
+
+- We find chapters with more than one organizers to be more successful, and therefore it
+  should be part of a minimum requirement.
+  
+- We don't have clear definition of what  or why one would want to start a chapter.
+  Generally, there is expectation for ongoing and regular community event, instead of
+  a one time workshop.
+ 
+- Each PyLadies chapter uses resources in many forms: they might have Pro meetup
+  account from The PSF, they may have website hosted, they might require funding from The PSF.
+
+- PyLadies admins received numerous inquiries from community members regarding local
+  chapters that turned out to be inactive, taking up additional resource/volunteer time
+  from Global leadership and administrator team to address these inquiries. 
+ 
+ - By better vetting new chapters, we hope to reduce unnecessary administrative work
+   down the road, and better focus our efforts in helping active chapters.
+
+## 2. What does it mean to start a PyLadies chapter?
+
+We should define the purpose of local PyLadies chapter, and why a chapter would be necessary.
+
+- help build and grow local Python community, through meetups, or sharing info through mailing lists, twitter, etc.
+
+What are the roles of a PyLadies chapter leader?
+
+- share news from global PyLadies leadership to their members
+
+- organize at least one meetup per year (?)
+
+- not necessarily be the one creating content/giving talk/workshop. 
+
+- provide report once a year to global PyLadies leadership 
+
+- notify global PyLadies leadership of changes in organizers (e.g. if they have new organizers, or if an organizer stepped down)
+
+### 3. What are minimum responsibilities and expectation when someone requested a PyLadies chapter? 
+
+- at least 2 co-organizers
+
+- preferably run by women
+
+- must be local (city-based). e..g Vancouver instead of BC or Canada
+
+- namespace must be descriptive (`kuala-lumpur` instead of `kl`)?
+
```

Comment:
I would add also that they have joined slack and have contacted another chapter for "starter mentorship" 

.. some of the chapters in Africa reached out to us with questions when they started.. I think it helps if people are explicitly aware that this is encouraged..

Reply:
the starter mentorship channel sounds great idea!

Reply:
I love this idea. Is "Starter Mentorship" going to be an official pairing or volunteer effort?

Reply:
At the moment I think official pairing will be difficult to organize. I was thinking to just start a #prospective-organizers channel on slack, it will be open so anyone can join and ask questions? Other experienced organizers can help and answer. What do you think?

Reply:
I would also go for that.. just to make it clear to people that help is available.. especially for the ones that are too shy to ask otherwise

Reply:
I've created the #prospective-organizers channel and have added an item to say they're advised to first join that channel. Please re-read and let me know if the phrasing can be improved.